File: 7days/Assets/CombineBlock.cs
using UnityEngine;
using System.Collections;

public class CombineBlock : MonoBehaviour
{

    // Use this for initialization
    void Start()
    {

    }

    // Update is called once per frame
    void Update()
    {
        //get_effect_search();
    }
    private void get_effect_search()
    {
        var games = GameObject.FindGameObjectsWithTag("EffectRange");
        foreach (var game in games)
        {
            if (game != null)
            {
                print(game);
                print(game.GetComponent<EffectRange>().get_effect_count());
            }
        }
    }
}

File: 7days/Assets/combine_block.js
#pragma strict

function Start () {

}

function Update () {

	get_effect_search();

}

function get_effect_search()
{
	var games = GameObject.FindGameObjectsWithTag( "effect_range" );
	
	for( var game in games )
	{
		if( game != null )
		{
			print(game);
			print(game.GetComponent( effect_range ).get_effect_count());
		}
	}	
}